tests/ha/single_pod_restart/test_server_pod_restart.py

```diff
@@ -163,20 +179,25 @@ def test_io_server_pod_restart(self):
                                                     log_prefix=self.test_prefix, skipcleanup=True)
         assert_utils.assert_true(resp[0], resp[1])
         LOGGER.info("Step 1: Performed WRITEs/READs/Verify with variable sizes objects")
-        LOGGER.info("Step 2: Shutdown random server pod by making replica=0 and verify cluster"
-                    " & remaining pods status")
+        num_replica = self.num_replica - 1
+        LOGGER.info("Step 2: Shutdown server pod by making replicas=0 and verify cluster & "
```

Comment:
we are not making replica 0 with sts method, so better just say shutdown server pod with replica method